Storage/tests/System/IamConfigurationTest.php

```diff
@@ -17,30 +17,23 @@
 
 namespace Google\Cloud\Storage\Tests\System;
 
+use Google\Cloud\Core\Exception\BadRequestException;
```

Comment:
Is there a unit test that should be updated in addition to this system test?

Reply:
In changes such as this where there's no code change to the library, just documentation and schema updates, we usually rely on system tests.